workshop.wwwapi/Endpoints/SurgeryEndpoint.cs

```diff
@@ -11,23 +15,201 @@ public static void ConfigurePatientEndpoint(this WebApplication app)
             var surgeryGroup = app.MapGroup("surgery");
 
             surgeryGroup.MapGet("/patients", GetPatients);
+            surgeryGroup.MapGet("/patients/{id}", GetAPatient);
+            surgeryGroup.MapPost("/patients/{id}", CreatePatient);
```

Comment:
why are you passing an ID in the create patient? the ID should be server created